### PR TITLE
fix toggle camera

### PR DESCRIPTION
**Pull request by ThyWoof**

### CODE REVIEW

File: SolastaCommunityExpansion/Models/GameUiContext.cs [hunk 1]
```diff
@@ -110,8 +110,10 @@ void ToggleZoomCamera()
                 }
                 else
                 {
-                    cameraService.DebugCameraEnabled = EnableDebugCamera;
+                    EnableDebugCamera = !EnableDebugCamera;
                 }
+
+                cameraService.DebugCameraEnabled = EnableDebugCamera;
```
Comment (ChrisPJohn) [inline SolastaCommunityExpansion/Models/GameUiContext.cs:116]: cameraService might be null

**Review by PhilPJL — APPROVED** [review]

**Review by PhilPJL — CHANGES_REQUESTED** [review]
Oops, should have read Chris's comment

File: SolastaCommunityExpansion/Models/GameUiContext.cs (hunk 1 above)
Comment (ThyWoof) [reply SolastaCommunityExpansion/Models/GameUiContext.cs]: fixed

File: SolastaCommunityExpansion/Models/GameUiContext.cs [hunk 2]
```diff
@@ -110,8 +110,9 @@ void ToggleZoomCamera()
                 }
                 else
                 {
+                    EnableDebugCamera = !EnableDebugCamera;
                     cameraService.DebugCameraEnabled = EnableDebugCamera;
-                }
+                }
```
Comment (PhilPJL) [inline SolastaCommunityExpansion/Models/GameUiContext.cs:115]: I think this should be 

```
...
else { EnableDebugCamera = !EnableDebugCamera; }
if(cameraService != null) { camera ... }
```

**Review by PhilPJL — CHANGES_REQUESTED** [review]
see comment

File: SolastaCommunityExpansion/Models/GameUiContext.cs (hunk 2 above)
Comment (PhilPJL) [reply SolastaCommunityExpansion/Models/GameUiContext.cs:115]: Although what function does IViewService have here?

Comment (ThyWoof) [reply SolastaCommunityExpansion/Models/GameUiContext.cs:115]: I copied this code from game and added the flag logic. Really don't know what role IViewService has in here.

Comment (PhilPJL) [reply SolastaCommunityExpansion/Models/GameUiContext.cs:115]: Don't trust the game code :)

Comment (ThyWoof) [reply SolastaCommunityExpansion/Models/GameUiContext.cs:110]: done

**Review by PhilPJL — APPROVED** [review]

**Pull request merged**